api/content/api.raml

```diff
@@ -0,0 +1,99 @@
+#%RAML 1.0 ResourceType
+title: Libero content API
+
+uses:
+  base: ../../core/api.raml
+
+types:
+    ID:
+        type: string
+        pattern: ^[A-Za-z0-9\-._]+$
+    Version:
+        type: LatestVersion | NumberVersion
+    LatestVersion:
+        type: string
+        enum: [latest]
+    NumberVersion:
+        type: integer
+        minimum: 1
+
+/{id}:
+    type:
+        base.item:
+            resource: Content item
+    description: |
+        Single content item.
+    uriParameters:
+        id:
+            description: |
+                Content item ID.
+            type: ID
+
+    delete:
+        description: |
+            Delete a content item.
+        responses:
+            202:
```

Comment:
Assumes these are using something like Airflow that actions it later. Should be `204` now?

Reply:
Can it support `20x` or `[202, 204]` to avoid assumptions?

Reply:
Yep. Airflow probably needs to be optional anyway.